common-services-api/pom.xml

```diff
@@ -12,10 +12,15 @@
     <packaging>bundle</packaging>
 
     <properties>
-      <vendor-driver.version>1.0.0.0</vendor-driver.version>
-      <pentaho-code.version>11.0.0.0</pentaho-code.version>
+      <org.osgi.core.version>4.3.1</org.osgi.core.version>
```

Comment:
Do not declare the osgi version here. Use the one from the parent pom: https://github.com/pentaho/maven-parent-poms/blob/master/pom.xml#L381-L386

Reply:
Replaced with parent pom's version